### ISSUE TITLE
(tvOS) tvOS 13 - not loading Cores

### ISSUE THREAD

**Issue by URBANsUNITED** [libretro/RetroArch#9397]
## Description

Retroarch on tvOS 13 is not able to load any roms/core. The rest of the menus are proper working incl. game pads for navigation.

### Actual behavior

Try to load content by choosing rom, but an error pop up "failed to load core"

### Steps to reproduce the bug

1. Choose load core
2. select one available core and press ok
3. nothing happs

1. Choose load content
2. choose any rom - ex. SNES rom and press ok
3. error pop up " failed to load core..."

### Version/Commit

- RetroArch: official stable 1.7.8 release 27.08.2019

### Environment information

Apple TV 4k
tvOS 13 beta 8 - next month is official release!
Signed with ios App signer - Dantheman - full DEV account

**Comment by RobLoach** [comment]
- Provide logs?
- Which core are you loading?

**Comment by yoshisuga** [comment]
I'm going to be looking at tvOS 13 support pretty soon - please be patient. The only way you can provide more information would be running via Xcode and indicating what the console output is. I suspect nothing is in the console because the dylib is not loading, and you'd need to attach a debugger to see what the issue is. 

I have a guess as to what the problem is - is that the cores probably need to be compiled using the tvOS SDK. The cores compiled with the iOS SDK seem to work on iOS 12, but perhaps not with iOS 13.

Again, I need to actually run this on an AppleTV to see and I just haven't had the chance to. It might take additional time to get the cores compiled with the tvOS SDK and more time to create pull requests for each core - something i'm not really looking forward to do.

**Comment by URBANsUNITED** [comment]
Hi!

Sorry, in my log files using Xcode is nothing I can find related to Retroarch.
And this problem belongs to all cores as it seams and yoshisuga already stated. None is working.

I want to help.
As I am no developer, but I have the device and other hardware and willing to test.
We can do one test with one core build with tvOS sdk only. So we know we are on the right way. 
Or you can show me how to do this by myself?!?

Cheers

URBANsUNITED

**Comment by yoshisuga** [comment]
Yep thats the right idea. I did compile some cores using tvOS SDK before,
and it worked - but it turned out that the iOS SDK cores did work at the
time so I abandoned the idea of creating a separate path for compiling for
tvOS.

You need to replace the SDK root to be the tvOS one and edit the build
scripts to do this. It's too much to explain so it's best I just do it :)

Thanks for offering to help - i can give you a link to an ipa once i have
it.

**Comment by yoshisuga** [comment]
I updated my AppleTV 4k to tvOS 13 beta 8, and I did confirm that cores did not start using the existing build.

I tweaked the build scripts and recompiled a couple cores (FCEUMM and bsnes) using the tvOS SDK and was able to get NES/Famicom and SNES/Super Famicom content to load.

I'll need to recompile all the cores for tvOS and create a new tvOS RetroArch ipa with the cores installed. It will take me a little time to get through all of them but at least something is going to work on tvOS 13.

**Comment by URBANsUNITED** [comment]
Great News!
Thanks for taking care and bringing light into this!
Now with this new conroller support we‘ll a fab plattform for retro gaming 😍

Cant wait for the first release🙊🙈
Thanks

**Comment by MDOBreno** [comment]
> I updated my AppleTV 4k to tvOS 13 beta 8, and I did confirm that cores did not start using the existing build.
> 
> I tweaked the build scripts and recompiled a couple cores (FCEUMM and bsnes) using the tvOS SDK and was able to get NES/Famicom and SNES/Super Famicom content to load.
> 
> I'll need to recompile all the cores for tvOS and create a new tvOS RetroArch ipa with the cores installed. It will take me a little time to get through all of them but at least something is going to work on tvOS 13.

Hello yoshisuga!

Where do/know about the download of your amazing cores that you recompiled for tvOS 13?

Thanks!

**Comment by MDOBreno** [comment]
I’m an iOS dev, and familiar with xcode. If you could make a video recompiling only once, I would be more than happy to do the entire process for all the cores for the community

**Comment by MDOBreno** [comment]
SOLVED!

I've read about compiling for tvOS, and successfully recompiled the most part of the cores for the tvOS 13 RetroArch, and here it is:

https://mega.nz/#!QlQB0ayb!3IETCIWtCunCz39sGapiPToFprLEtvSruXZZLkCKfjs
